.circleci/config.yml

```diff
@@ -243,7 +243,6 @@ jobs:
                   keys:
                       - v2-golden-images-<< pipeline.parameters.current_golden_images_hash >>-hcm-
                       - v2-golden-images-main-hcm-
-                      - v2-golden-images-main-spectrum-lightest-medium-ltr-
```

Comment:
Why did we remove it?

Reply:
We removed the `lightest` tests, so they are no longer part of our test suite. As a result, we will not be regenerating the `v2-golden-images-main-spectrum-lightest-medium-ltr-` cache key. While we could change it to `light`, these are fallback cache keys and are in order. The first two should be sufficient. These [caches stay-alive time is 15 days on CircleCI](https://app.circleci.com/settings/plan/github/adobe/usage-controls).